set.mm

```diff
@@ -500210,6 +500210,53 @@ have become an indirect lemma of the theorem in question (i.e. a lemma
       DUGUEBAIZMUIBEAPUJUFUEBAQRSTUAABCDEFGUBUC $.
   $}
 
+  ${
+    $d I x $.  $d P k x $.  $d F k x $.  $d G k x $.
+    revwlk.1 $e |- V = ( Vtx ` G ) $.
+    revwlk.2 $e |- I = ( iEdg ` G ) $.
```

Comment:
These hypothesies are not necessary, because V and I do not appear in the statement.

Reply:
Therefore, they sould be removed (`V` and `I` replaced by ``( Vtx ` G )`` and ``( iEdg ` G )`` in the proof).

Reply:
Done, thanks!